slack_bolt/request/payload_utils.py

```diff
@@ -182,6 +182,15 @@ def to_step(body: Dict[str, Any]) -> Optional[Dict[str, Any]]:
     return None
 
 
+# -------------------
+# Function
+# -------------------
+
+
+def is_function_interactivity(body: Dict[str, Any]) -> bool:
+    return "function_data" in body
```

Comment:
Technically the presence of `function_data` in the payload body can signify _either_ interactivity events (block_actions, view_submission, view_closed) but it could ALSO signal the function-executed event. Not sure if that's relevant here.

Reply:
Good catch will add more validations here seems like bolt validates the type as well 🥇